Tipstaff/Mappers/IMapperCollections.cs

```diff
@@ -4,6 +4,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using Tipstaff.Models;
+using Tipstaff.Presenters;
 using Tipstaff.Services.DynamoTables;
 
 namespace Tipstaff.Mappers
```

Comment:
this comment is only cause I've just noticed that DACP uses the IMappers from TPLibrary - how come Tipstaff has it's own set of IMapper?